unit_tests/test_tlslite_keyexchange.py

```diff
@@ -1247,6 +1247,29 @@ def test_ECDHE_key_exchange_with_invalid_CKE(self):
         with self.assertRaises(TLSIllegalParameterException):
             self.keyExchange.processClientKeyExchange(cln_key_ex)
 
+    def test_ECDHE_key_exchange_with_empty_value_in_CKE(self):
+        srv_key_ex = self.keyExchange.makeServerKeyExchange('sha1')
+
+        KeyExchange.verifyServerKeyExchange(srv_key_ex,
+                                            self.srv_pub_key,
+                                            self.client_hello.random,
+                                            self.server_hello.random,
+                                            [(HashAlgorithm.sha1,
+                                              SignatureAlgorithm.rsa)])
+
+        curveName = GroupName.toStr(srv_key_ex.named_curve)
+        curve = getCurveByName(curveName)
+        generator = curve.generator
+        cln_Xc = ecdsa.util.randrange(generator.order())
+        cln_Ys = decodeX962Point(srv_key_ex.ecdh_Ys, curve)
+        cln_Yc = encodeX962Point(generator * cln_Xc)
```

Comment:
What is the point of this block? I don't see it used later and it is just copy&paste from the previous, where the `cln_Yc` is used for `createECDH()`.